src/writing-your-own-application/index.md

```diff
@@ -2,11 +2,14 @@
 
 With the appropriate Rust compiler and toolchain installed, you're now ready to create an application.
 
-There are essentially two ways to do this: generating from a template or starting from scratch using only `cargo`.
+You can write an application in the following ways:
 
-We strongly recomend starting projects from templates as it gives you a configured project saving all the time that would require
-to setup a project started from scratch with `cargo`.
+- Generate from a template: (**Strongly recommended**). Gives you a configured project, saves time and prevents possible errors.
+- Start from scratch using Cargo: Requires more expertise since you need to configure several parts of the project.
 
-> If you are looking for inspiration, check the [Projects section of our Awesome ESP Rust repository]
 
-[Projects section of our Awesome ESP Rust repository]: https://github.com/esp-rs/awesome-esp-rust#projects
+This chapter won't cover the instructions on how to create a project from scratch with `cargo`, it will only focus on generating a project from template project.
```

Comment:
> Are there any instructions/suggestions on how to create a project from scratch and, if yes, where?

No, there are not and I dont think we should do them, we have the template, if an user wants to start from scratch they are on their own.

> cargo -> Cargo?

I haven't been updating the format, IIRC, we decided to update the format once we have the guidelines completed and all at once?

Reply:
Ah OK, then I think it makes sense to say something to the effect that you should use the template and don't bother with the standard way.

Otherwise, too much attention is given to the standard way and it comes across as something useful.